src/Prelude.hs

```diff
@@ -43,3 +47,6 @@ import qualified Colog (Message, WithLog)
 
 -- | 'Colog.WithLog' alias specialized to 'Message' data type.
 type WithLog env m = Colog.WithLog env Colog.Message m
+
+julianDayToIso :: Day -> Text
+julianDayToIso = fromString . formatTime defaultTimeLocale (iso8601DateFormat Nothing)
```

Comment:
Again, this function is not going to be used in every module. So it's better to put it into a separate module like `IW.Time`